.travis.yml

```diff
@@ -24,6 +24,8 @@ matrix:
       env: DEPENDENCIES='low'
     - php: 5.6
       env: DEPENDENCIES='dev'
+    - php: 7.0
+      env: DEPENDENCIES='dev'
```

Comment:
no need to test dev dependencies on both 5.6 and 7.0 IMO. Please remove this extra job